package.json

```diff
@@ -18,16 +18,16 @@
     "start": "ui5 serve --open index.html"
   },
   "dependencies": {
-    "@openui5/sap.f": "^1.60.0",
-    "@openui5/sap.m": "^1.60.0",
-    "@openui5/sap.ui.core": "^1.60.0",
-    "@openui5/sap.ui.layout": "^1.60.0",
-    "@openui5/themelib_sap_belize": "^1.60.0",
+    "@openui5/sap.f": "^1.60.1",
+    "@openui5/sap.m": "^1.60.1",
+    "@openui5/sap.ui.core": "^1.60.1",
+    "@openui5/sap.ui.layout": "^1.60.1",
+    "@openui5/themelib_sap_belize": "^1.60.1",
     "openui5-camera": "~0.0.11",
     "openui5-chartjs": "https://github.com/StErMi/openui5-chartjs",
     "openui5-flatpickr": "https://github.com/StErMi/openui5-flatpickr",
     "openui5-googlemaps": "https://github.com/jasper07/openui5-googlemaps",
-    "openui5-model-json-crud": "^0.0.6",
+    "openui5-model-json-crud": "0.0.6",
```

Comment:
your lib is already at 0.08 - don't you want the latest version to be consumed here?

Reply:
How did I miss that??? Done.